Mathlib/Data/Set/Operations.lean

```diff
@@ -228,6 +228,9 @@ end Pi
 /-- Two functions `f₁ f₂ : α → β` are equal on `s` if `f₁ x = f₂ x` for all `x ∈ s`. -/
 def EqOn (f₁ f₂ : α → β) (s : Set α) : Prop := ∀ ⦃x⦄, x ∈ s → f₁ x = f₂ x
 
+lemma EqOn_comm {f₁ f₂ : α → β} {s : Set α} : EqOn f₁ f₂ s ↔ EqOn f₂ f₁ s := by
```

Comment:
We already have `eqOn_comm`.

Reply:
I thought it was strange we didnt have it! but loogle came back with no results. I've removed it now.